src/main/java/seedu/uninurse/logic/commands/ClearCommand.java

```diff
@@ -2,8 +2,11 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import seedu.uninurse.model.Model;
-import seedu.uninurse.model.UninurseBook;
+import seedu.uninurse.model.person.Patient;
 
 /**
  * Clears the uninurse book.
```

Comment:
Currently, the message for clear is "UniNurse Book has been cleared!". Perhaps we should also change the message to something like "Listed patients have been cleared!"?

Reply:
Done